Add a bibliography to the GPflow documentation. (#1842)
GPflow/GPflow: doc/sphinx/intro.rst

------------

GPflow is a package for building Gaussian process models in python, using `TensorFlow <http://www.tensorflow.org>`_. It was originally created and is now managed by `James Hensman <http://www.lancaster.ac.uk/staff/hensmanj/>`_ and `Alexander G. de G. Matthews <http://mlg.eng.cam.ac.uk/?portfolio=alex-matthews>`_.

> The full list of `contributors <http://github.com/GPflow/GPflow/graphs/contributors>`_ (in alphabetical order) is Alexander G. de G. Matthews, Alexis Boukouvalas, Artem Artemev, Daniel Marthaler, David J. Harris, Hugh Salimbeni, Ivo Couckuyt, James Hensman, Keisuke Fujii, Mark van der Wilk, Mikhail Beck, Pablo Leon-Villagra, Rasmus Bonnevie, ST John, Tom Nickson, Valentine Svensson, Vincent Dutordoir, Zoubin Ghahramani. GPflow is an open source project so if you feel you have some relevant skills and are interested in contributing then please do contact us.

Reviewer: ```suggestion
We maintain a [full list of past contributors](https://github.com/GPflow/GPflow/blob/develop/CONTRIBUTORS.md). GPflow is an open source project so if you feel you have some relevant skills and are interested in contributing then please do contact us.
```
This is out of date, and the linked github graph is incomplete (can explain why if you care).

approximations for `multioutput GPs <notebooks/advanced/multioutput.html>`_.

GPflow has a unifying design for using multioutput GPs and specifying interdomain approximations. A review of the
mathematical background and the resulting software design is described in :cite:t:`GPflow2020multioutput`.

GPLVM
"""""

> For visualisation, the GPLVM :cite:p:`lawrence2003gaussian` and Bayesian GPLVM `titsias2010bayesian` models are implemented

Reviewer 2: Is the 'titsias2010bayesian' missing a 'cite'?
